server/src/test/java/org/eclipse/lsp/cobol/usecases/TestSemicolonAllowedOnNotTerminatingPositions.java

```diff
@@ -23,8 +23,8 @@
 import org.eclipse.lsp4j.DiagnosticSeverity;
 import org.junit.jupiter.api.Test;
 
-/** This test checks that the semicolons are not ignored in the unexpected positions. */
-class TestSemicolonNotAllowedOnNotTerminatingPositions {
+/** This test checks that the semicolons followed by space are considered as separators just like space. */
+class TestSemicolonAllowedOnNotTerminatingPositions {
```

Comment:
This is a valid syntax and the program compiles fine.
Doc ref - https://www.ibm.com/docs/en/cobol-zos/6.3?topic=separators-rules